packages/react-ui-core/src/Carousel/Carousel.js

```diff
@@ -24,6 +24,7 @@ const carouselItems = PropTypes.arrayOf(
 export default class Carousel extends Component {
   static propTypes = {
     className: PropTypes.string,
+    'data-tid': PropTypes.string,
```

Comment:
this feels like something that could be optionally spread in from the the `...rest` below - any thoughts?

Reply:
The spread below is passed directly into the Slider and the moment it's destructed to include in the Slider's parent, it has to be declared as a PropType.